classes/requests/helpers/class-dintero-checkout-order.php

```diff
@@ -430,7 +430,8 @@ public function get_shipping_object() {
 			 *
 			 * @var WC_Order_Item_Shipping $shipping_line The shipping line.
 			 */
-			$shipping_line = array_values( $shipping_lines )[0];
+			$shipping_line    = array_values( $shipping_lines )[0];
+			$shipping_line_id = $this->order->get_meta( '_dintero_shipping_line_id' ) ?? $shipping_line->get_method_id() . ':' . $shipping_line->get_instance_id();
```

Comment:
The `??` operator only checks against null values, and since `WC_Order::get_meta` defaults to an empty string as a fallback if no value exists, it will never be able to get to the fallback here. I would recommend doing an `empty()` check instead.